### PR TITLE
Remove layer,image, container code that is not used

### PR DESCRIPTION
**Pull request by rhatdan**
Lots of cut and pasted code that ends up doing nothing, but takes time
to process.
Signed-off-by: Dan Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: store.go [hunk 1]
```diff
@@ -692,12 +677,6 @@ func (s *store) CreateImage(id string, names []string, layer, metadata string, o
 	if modified, err := ristore.Modified(); modified || err != nil {
 		ristore.Load()
 	}
-	rcstore.Lock()
-	defer rcstore.Unlock()
-	defer rcstore.Touch()
```
Comment (TomSweeneyRedHat) [inline store.go]: Ditto slight Touch removal concern here too.

**Comment by TomSweeneyRedHat** [comment]
LGTM, but would like to get someone with more familiarity to eyeball, especially the touch removals.  Overall a nice cleanup.

File: store.go [hunk 2]
```diff
@@ -628,12 +624,6 @@ func (s *store) PutLayer(id, parent string, names []string, mountLabel string, w
 	if modified, err := rlstore.Modified(); modified || err != nil {
 		rlstore.Load()
 	}
-	ristore.Lock()
-	defer ristore.Unlock()
-	defer ristore.Touch()
```
Comment (TomSweeneyRedHat) [inline store.go]: I think this removal is OK too, but don't know for sure.  I'm just not sure if the Touch should be removed here.

**Comment by rhatdan** [comment]
@nalind PTAL

File: store.go (hunk 2 above)
Comment (nalind) [inline store.go]: We're okay not calling Touch() if the set of data that the store object keeps isn't being changed, which appears to be true here.

**Comment by nalind** [comment]
LGTM.

**Pull request merged**